framework/src/org/checkerframework/framework/util/defaults/QualifierDefaults.java

```diff
@@ -105,6 +105,16 @@
     private final Map<Element, Boolean> elementAnnotatedFors = new IdentityHashMap<>();
 
     /**
+     * CLIMB-to-the-top locations whose standard default is top
+     */
+    public static final DefaultLocation[] standardClimbDefaultsTop = { DefaultLocation.LOCAL_VARIABLE, DefaultLocation.RESOURCE_VARIABLE,
```

Comment:
So I had incorrectly assumed that "CLIMB-to-the-top" are only the locations that are flow-sensitively refined and forgot that "IMB" is for implicit bounds.
Local variables and resource variables are refined and therefore top.
Implicit bounds take the top (upper) and bottom (lower) qualifiers.
In most type systems exception parameters are top, unless the JLS gives us some property of the parameter, e.g. that the object is non-null.
I'm not sure "CLIMB-to-the-top" summarizes this well.